### PR TITLE
Feat booking confirmation mail

### PR DESCRIPTION
**Pull request by ImMPrada**
This PR enables the Booking Flights app, to send booked flight confirmation emails to passengers once the Booking is committed at the db

It uses ActiveJob

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -64,6 +64,7 @@ group :development, :test do
   gem 'rubocop-rspec'
   gem 'shoulda-matchers', '~> 5.0'
   gem 'simplecov'
+  gem "letter_opener"
```
Comment (ImMPrada) [inline Gemfile:67]: This gem opened the send email at the browser

File: app/controllers/bookings_controller.rb [hunk 2]
```diff
@@ -11,7 +11,10 @@ def new
 
   def create
     booking_creator.build
-    redirect_to booking_path(booking_creator.commit)
+    booking = booking_creator.commit
+    redirect_to booking_path(booking)
+
+    SendBookingMailToPassengerJob.perform_later(booking)
```
Comment (ImMPrada) [inline app/controllers/bookings_controller.rb]: BookingCreatr#commit returns the db committed booking

File: app/jobs/send_booking_mail_to_passenger_job.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+class SendBookingMailToPassengerJob < ApplicationJob
+  def perform(booking)
+    booking.passengers.each do |passenger|
+      BookingMailer.flight_booked(booking, passenger).deliver_later
+    end
+  end
+end
```
Comment (ImMPrada) [inline app/jobs/send_booking_mail_to_passenger_job.rb]: The job enqueue the confirmation emails to the passengers

**Review by ImMPrada — COMMENTED** [review]
some notes...

File: app/jobs/send_booking_mail_to_passenger_job.rb (hunk 3 above)
Comment (mapra99) [reply app/jobs/send_booking_mail_to_passenger_job.rb]: I'd suggest calling this `SendBookingMailToPassengersJob`, with passengers in plural. Or `SendBookingMails`

**Review by mapra99 — APPROVED** [review]

**Pull request merged**